util/encoding/encoding.go

```diff
@@ -260,25 +259,25 @@ func DecodeVarint(b []byte) ([]byte, int64) {
 	length := int(b[0]) - 8
 	if length < 0 {
 		length = -length
-		b = b[1:]
-		if len(b) < length {
-			panic(fmt.Sprintf("insufficient bytes to decode var uint64 int value: %s", b))
+		remB := b[1:]
```

Comment:
i had to change it on 276 because it would otherwise shadow, and then decided to stay consistent in this branch